kas.yml

```diff
@@ -1,11 +1,6 @@
 header:
   version: 8
 
-distro: pce-xen
```

Comment:
@Asiderr README should be updated in this PR. building kas.yml will no longer work with this change.